### PR TITLE
Add Jiaxin to the leaderboard

### PR DESCRIPTION
**Pull request by claire-fang**
Final validation loss: 3.32098
Learning curve: https://api.wandb.ai/links/jiaxinf-stanford-university/gm6mft4d
Description:
Model config: d_model=768, num_layers=8, num_heads=12, d_ff=2048, context_length=1024, vocab_size=32000
Optimizer: AdamW with max_lr=3e-3, min_lr=3e-4, warmup_steps=500, total_steps=12000, weight_decay=0.1, β1=0.9, β2=0.95, grad_clip=1.0, batch_size=128
Training optimizations:
torch.compile for graph-level optimization
torch.autocast with bfloat16 for mixed-precision training
optimizer.zero_grad(set_to_none=True) for reduced memory overhead

Trained on a single B200 in ~45 minutes.

### CODE REVIEW

**Comment by hermabr** [comment]
Thanks! Please update the pr so that your score is in the right location:))

**Comment by claire-fang** [comment]
Just fix it! Thanks for the catch :)
